requirements-testing.txt

```diff
@@ -1,3 +1,6 @@
 click
 mistune
 tox
+sphinx
```

Comment:
I would just keep these in `requirements.txt`, since they are required to actually build the docs.